src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -105,6 +105,7 @@ public class CaseData extends CaseDataParent implements MappableObject {
     private final GAStatementOfTruth generalAppStatementOfTruth;
     private final GAHearingDetails generalAppHearingDetails;
     private final GASolicitorDetailsGAspec generalAppApplnSolicitor;
+    private final String generalAppSuperClaimType;
```

Comment:
why do we need this field? It is neither being set any value nor being used anywhere?

Reply:
After talking to Smita - I deleted these values as they were not needed under CaseData. Tested again works approx.